include/Surelog/Utils/NumUtils.h

```diff
@@ -25,123 +25,167 @@
 #define SURELOG_NUMUTILS_H
 #pragma once
 
+#include <cctype>
+#include <charconv>
 #include <cstdint>
 #include <string>
-#include <charconv>
 #include <string_view>
-#include <ctype.h>
 
 namespace SURELOG {
+class NumUtils final {
```

Comment:
I suggest to actually get _rid_ of the NumUtils class as using a static class just as a namespace is a bad pattern that we should get rid of.

So if you want these in a NumUtils namespace, create a namespace NumUtils (and also fix the previous NumUtils class).

Reply:
Using a class as container of static functions is an antipattern that we should avoid and in particular not perpetuate further. Having an internal namespace is the lesser evil.